packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/dedicatedCloud-datacenter-drp.service.js

```diff
@@ -398,23 +398,30 @@ export default class {
     );
   }
 
-  deleteZertoOptionOrderInUserPref(drpKey) {
-    return this.ovhUserPref.remove(drpKey, true);
+  deleteZertoOptionOrderInUserPref(pccId) {
+    const preferenceKey = this.constructor.formatPreferenceKey(
+      pccId,
+      DEDICATEDCLOUD_DATACENTER_ZERTO.title,
+    );
+
+    return this.ovhUserPref.remove(preferenceKey, true);
   }
 
   /* ------- Order ZERTO option ------ */
 
   disableDrp(drpInformations) {
-    const deleteDrpPromise =
-      drpInformations.drpType === DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS.ovh
-        ? this.disableDrpOvh(drpInformations)
-        : this.disableDrpOnPremise(drpInformations);
-
     const drpPccId = drpInformations.primaryPcc.serviceName;
 
-    return deleteDrpPromise
-      .then(() => this.deleteZertoOptionOrderInUserPref(drpPccId))
-      .then(() => this.deleteDisableSuccessAlertPreference(drpPccId));
+    return (drpInformations.drpType ===
+    DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS.ovh
+      ? this.disableDrpOvh(drpInformations)
+      : this.disableDrpOnPremise(drpInformations)
+    ).then(() =>
+      Promise.allSettled([
```

Comment:
⚠️ Using `Promise.allSettled` isn't supported in both **Edge** and **Internet Explorer**
Browser.

`babel-polyfill` relates to `core-js/shim` and brings us some useful polyfills
to help us to support a large range of Browser vendors but the version
of `core-js/shim` (locked in the `yarn.lock`) is:

```sh
$ yarn why core-js | grep "Found"
=> Found "core-js@2.6.10"
```

After reviewing the source of `core-js` package, I wasn't able to find the
corresponding polyfill to `Promise.allSettled` so I've spend more time to search
in their repository then I came to this following Issue:
- https://github.com/zloirock/core-js/issues/399#issuecomment-395310742

So to support this we have to bump `core-js` to version **^3.0.0**

This changes should be addressed in a separate Pull Request because we must
consolidate if we don't introduce some regressions.

IMO it must be addressed in a separate Pull Request and we must spend 

Alternative:

Why not considering using this dependency: https://github.com/ovh-ux/ng-q-allsettled?

Resources:
- https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Promise/allSettled
- https://github.com/zloirock/core-js/issues/399